pkg/apis/application/v1alpha1/types.go

```diff
@@ -315,6 +315,9 @@ type ApplicationSourceKustomize struct {
 	Images KustomizeImages `json:"images,omitempty" protobuf:"bytes,3,opt,name=images"`
 	// CommonLabels adds additional kustomize commonLabels
 	CommonLabels map[string]string `json:"commonLabels,omitempty" protobuf:"bytes,4,opt,name=commonLabels"`
+	// ForceCommonLabels force overwrites commonLabel directive if it
+	// already exists within the Kustomize application manifests
+	ForceCommonLabels bool `protobuf:"varint,6,opt,name=forceCommonLabels"`
```

Comment:
This should be moved to the bottom of the struct to keep order with the # of the field in the protobuf message.

The new field also lacks the `json` annotation, which is why `gen-crd-spec` panics.

Reply:
Thanks, that worked!